subset_vds.py

```diff
@@ -176,6 +176,24 @@ def convert_intervals_to_locus(interval_list: list[str]) -> list[IntervalExpress
     return locus_list
 
 
+def parse_samples_list(samples: str) -> list[str]:
+    """read samples from either the command line or a file
+
+    Args:
+        samples (str): a string of either comma separated samples, or a path to a file of samples (one per line) in gcs
+
+    Returns:
+        list[str]: a list of strings representing samples
+    """
+    sample_list: list[str] = []
+    if to_path(samples).exists():
+        with open(to_path(samples), "rt") as samples_file:
```

Comment:
Tiny change if you want
`    if (sample_file := to_path(samples)).exists():
        with open(sample_file, "rt") as samples_file:`

Reply:
@Alexander-Stuckey loves a good walrus